content/test/unit/storage/MockedStorage.ts

```diff
@@ -2,8 +2,15 @@ import { Readable } from 'stream'
 import { ContentItem, ContentStorage, SimpleContentItem, streamToBuffer } from '../../../src/storage/ContentStorage'
 
 export class MockedStorage implements ContentStorage {
-  private storage: Map<string, Buffer> = new Map()
+  private storage: Map<string, Uint8Array> = new Map()
 
+  size(fileHash: string): Promise<number | undefined> {
+    const buffer = this.storage.get(fileHash)
+    return Promise.resolve(buffer?.byteLength)
+  }
+  storeContent(fileHash: string, content: Uint8Array | Readable) {
+    return Promise.resolve()
```

Comment:
Just want to make sure this noop Promise.resolve is intended. Asking because storeStream does store the stream, it seems trivial to add the actual beahvior here

```suggestion
    this.storage.set(id, content instanceof Uint8Array ? content : await streamToBuffer(content))
```